decorator.rb

```diff
@@ -0,0 +1,23 @@
+require './nameable'
+
+class Decorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
```

Comment:
- [ ] As part of the requirements , the Decorator class should inherit from Namable kindly update your class , and also make sure to define :nameable as attr_accessor  🙏 

example : 
```ruby
class Decorator < Nameable
    attr_accessor :nameable
  
    def initialize(nameable)
      @nameable = nameable
    end
  

    def correct_name
      @nameable.correct_name
    end
end
```